blocks/library/latest-posts/index.js

```diff
@@ -33,6 +33,8 @@ registerBlockType( 'core/latest-posts', {
 
 	category: 'widgets',
 
+	keywords: [ __( 'recent-posts' ) ],
```

Comment:
I think if you search for `tagB` it won't work. but it's the same if it was `tagA-tagB`